fares-data-build-tool/create-data
# feat: Edit operator group services

## Description

Feature to allow the user to edit the services they have selected for their operator group. Also some refactoring and logging added.

## Testing instructions

Create an operator group, and use it to make a multi operator ticket. Then edit the services you selected for the additional operators, and ensure it has updated. Also, test around the operator group area in general as there was a lot of refactoring.
